STAT697-01_s20-team-2_data_preparation.sas

```diff
@@ -429,9 +403,9 @@ enrollment per county, and the average value of Estimated FTE per county. */
 proc sql;
```

Comment:
Starting from this line of query, I was trying to combine three datasets (gradaf17, enr16, Staffassign16)  into one table.